map/apply-scheduled-updates-to-preplanned-map-area/src/main/java/com/esri/samples/apply_scheduled_updates_to_preplanned_map_area/ApplyScheduledUpdatesToPreplannedMapAreaSample.java

```diff
@@ -0,0 +1,274 @@
+import java.util.concurrent.ExecutionException;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.Job;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Expiration;
+import com.esri.arcgisruntime.mapping.MobileMapPackage;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.tasks.geodatabase.SyncGeodatabaseParameters;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncJob;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncParameters;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncResult;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncTask;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapUpdatesInfo;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineUpdateAvailability;
+import com.esri.arcgisruntime.tasks.offlinemap.PreplannedScheduledUpdatesOption;
+
+import org.apache.commons.io.FileUtils;
+
+public class ApplyScheduledUpdatesToPreplannedMapAreaSample extends Application {
+
+  private MapView mapView;
+  private MobileMapPackage mobileMapPackage;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Apply Schedueld Updates to Preplanned Map Area");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map view
+      mapView = new MapView();
+
+      // create progress indicator
+      ProgressIndicator progressIndicator = new ProgressIndicator();
+      progressIndicator.setVisible(false);
+
+      // create a button to update the offline map
+      Button applyUpdatesButton = new Button("Apply Scheduled Updates");
+      applyUpdatesButton.setDisable(true);
+
+      // create labels to show update availability and size
+      Label updateAvailableLabel = new Label("Updates: ");
+      updateAvailableLabel.setTextFill(Color.WHITE);
+      Label updateSizeLabel = new Label("Update size: ");
+      updateSizeLabel.setTextFill(Color.WHITE);
+
+      // create a control panel for the UI elements
+      VBox controlsVBox = new VBox(6);
```

Comment:
on a Mac, the control panel is too narrow and the labels and button are cut off (this has happened to me before with other samples, must be a different resolution involved). Shouldn't have to be too much wider, enough to fit in 4 extra characters.